src/xenia/app/emulator_window.cc

```diff
@@ -928,6 +936,41 @@ void EmulatorWindow::ToggleFullscreen() {
   SetFullscreen(!window_->IsFullscreen());
 }
 
+void EmulatorWindow::TakeScreenshot() {
+  xe::ui::RawImage myimage;
+  if (emulator()->is_title_open()) {  // Make sure we have a game open first
+    GetGraphicsSystemPresenter()->CaptureGuestOutput(myimage);
```

Comment:
The presenter is not initialized immediately after the window is opened — see how the call to `SetupGraphicsSystemPresenterPainting` is done. Therefore, for some time (possibly seconds) after opening the emulator, `GetGraphicsSystemPresenter()` may return `nullptr`, and trying to take a screenshot may result in a crash due to dereferencing a null pointer.

Reply:
That's fine, we can just check to make sure that `GetGraphicsSystemPresenter()` isn't a nullptr. If it is, we can log the error.